save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/OrganizationView.kt

```diff
@@ -523,14 +529,14 @@ class OrganizationView : AbstractView<OrganizationProps, OrganizationViewState>(
                     }
                     img {
                         className = ClassName("avatar avatar-user width-full border color-bg-default rounded-circle")
-                        src = state.organization?.avatar?.let {
-                            "/api/$v1/avatar$it"
-                        }
-                            ?: run {
-                                "img/company.svg"
-                            }
+                        src = state.avatar
                         height = 100.0
                         width = 100.0
+                        onError = {
+                            setState {
+                                avatar = "img/undraw_image_not_found.png"
```

Comment:
maybe extract this variable to a separate constant?

Reply:
Added